.vib/common/goss/templates/check-directories.yaml

```
@@ -6,19 +6,23 @@
 #   - .Vars.directories.paths (Array)
 # Optional vars:
 #   - .Vars.directories.mode (String)
+#   - .Vars.directories.owner (String)
+#   - .Vars.directories.group (String)
 ########################
 file:
-  {{ range .Vars.directories }}
-  {{ $mode := "" }}
-  {{ if contains "mode:" (. | toString) }}
-  {{ $mode = .mode }}
-  {{ end }}
-  {{ range $directory := .paths }}
-  {{ $directory }}:
+  {{ range $dir := .Vars.directories }}
```

I would personally rename this var as it conveys the idea that an entry of `directories` is a single directory, when in reality each entry may have multiple directories (paths).

```suggestion
  {{ range $dirs := .Vars.directories }}
```